rustworkx-core/src/generators/random_graph.rs

```diff
@@ -12,6 +12,7 @@
 
 #![allow(clippy::float_cmp)]
 
+use core::f64;
```

Comment:
I think you can just use f64 out of the box? We always include `std`, I think we never really use core